update.sh

```diff
@@ -152,6 +151,12 @@ for i in `awk '/SOURCES|HEADERS/ { print $3 }' crypto/Makefile.am` ; do
 	fi
 done
 
+for i in $libcrypto_src/arch/*; do
+	arch=`basename $i`
+	mkdir -p include/arch/$arch
+	$CP $libcrypto_src/arch/$arch/opensslconf.h include/arch/$arch/
```

Comment:
Maybe I'm missing something, but this doesn't look right. `opensslconf.h` needs to end up in `include/openssl` as public headers pull it in.

Reply:
That's because I had all the rest of the changes locally and had put them in the wrong file, Makefile.am, when they should have been in Makefile.am.tpl. Updated and committed the correct file here: https://github.com/libressl/portable/commit/c3e1e50cb19621ec5bcea424868f8e70fde60cdf